src/interfaces/IProposalGenericExecutor.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @dev Generic proposal interface allowing execution via MockExecutor
+ */
+interface IProposalGenericExecutor {
+  function execute() external;
+}
```

Comment:
This is just used for testing (and will probably be removed in the next PR). I would inline it in the `PayloadWithEmit` for now so we don't forget to remove later.

Reply:
Done ✅